tests/components/deconz/test_light.py

```diff
@@ -48,35 +52,665 @@ async def test_no_lights_or_groups(hass, aioclient_mock):
     assert len(hass.states.async_all()) == 0
 
 
-async def test_lights_and_groups(hass, aioclient_mock, mock_deconz_websocket):
+@pytest.mark.parametrize(
+    "input,expected",
+    [
+        (  # RGB light in color temp color mode
+            {
+                "colorcapabilities": 31,
+                "ctmax": 500,
+                "ctmin": 153,
+                "etag": "055485a82553e654f156d41c9301b7cf",
+                "hascolor": True,
+                "lastannounced": None,
+                "lastseen": "2021-06-10T20:25Z",
+                "manufacturername": "Philips",
+                "modelid": "LLC020",
+                "name": "Hue Go",
+                "state": {
+                    "alert": "none",
+                    "bri": 254,
+                    "colormode": "ct",
+                    "ct": 375,
+                    "effect": "none",
+                    "hue": 8348,
+                    "on": True,
+                    "reachable": True,
+                    "sat": 147,
+                    "xy": [0.462, 0.4111],
+                },
+                "swversion": "5.127.1.26420",
+                "type": "Extended color light",
+                "uniqueid": "00:17:88:01:01:23:45:67-00",
+            },
+            {
+                "entity_id": "light.hue_go",
+                "state": STATE_ON,
+                "attributes": {
+                    ATTR_BRIGHTNESS: 254,
+                    ATTR_COLOR_TEMP: 375,
+                    ATTR_EFFECT_LIST: [EFFECT_COLORLOOP],
+                    ATTR_SUPPORTED_COLOR_MODES: [
+                        COLOR_MODE_COLOR_TEMP,
+                        COLOR_MODE_HS,
+                        COLOR_MODE_XY,
+                    ],
+                    ATTR_COLOR_MODE: COLOR_MODE_COLOR_TEMP,
+                    ATTR_MIN_MIREDS: 153,
+                    ATTR_MAX_MIREDS: 500,
+                    ATTR_SUPPORTED_FEATURES: 44,
```

Comment:
Consider:
`
ATTR_SUPPORTED_FEATURES: SUPPORT_TRANSITION | SUPPORT_FLASH | SUPPORT_EFFECT,
`

Reply:
Nice! I will do a small follow up PR with that

Reply:
Fixed in #52096